patterns/1-initial/release-process.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+Standard Release Process/Published Artifacts
```

Comment:
```suggestion
Standard Release Process and Published Artifacts
```

This pattern is advocating for both of these things.
I think that this would become more clear this way.